assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -59,7 +59,7 @@ data "ct_config" "ignitions" {
       cluster_domain_suffix = var.cluster_domain_suffix
       node_labels = merge({
         "node.kubernetes.io/node"                 = "",
-        "lokomotive.alpha.kinvolk.io/bgp-enabled" = format("%t", !var.disable_bgp),
+        "lokomotive.alpha.kinvolk.io/bgp-enabled" = format("%t", ! var.disable_bgp),
```

Comment:
What version of Terraform you use? For me, 0.14.x allows `!var.x` formatting.

Reply:
It seems GitHub Action agrees with me :smile:

Reply:
I believe the `terraform fmt` should be based on 0.13. Github Actions needs to change.
What do you think ?

Reply:
Not sure how to proceed here :thinking:

Reply:
I'm not even sure why GitHub Action has Terraform already installed actually :smile: But yeah, we can change GitHub action to have 0.13.x then.